src/app/components/AddDocumentForm/AddDocumentForm.module.css

```diff
@@ -14,3 +14,14 @@
   color: var(--color-text);
   background-color: var(--color-tertiary);
 }
+
+.documentSave:hover {
+  cursor: pointer;
+}
+
+.documentSave:disabled,
+button[disabled] {
```

Comment:
Be careful using element selectors. This effects all `progress` in your project.